src/NLU.DevOps.CommandLine/Compare/CompareCommand.cs

```diff
@@ -33,7 +33,7 @@ public static int Run(CompareOptions options)
 
             if (options.Metadata)
             {
-                var expectedUtterances = Read<List<LabeledUtterance>>(options.ExpectedUtterancesPath);
+                var expectedUtterances = Read<List<CompareLabeledUtterance>>(options.ExpectedUtterancesPath);
                 var actualUtterances = Read<List<ScoredLabeledUtterance>>(options.ActualUtterancesPath);
                 var compareResults = TestCaseSource.GetNLUCompareResults(expectedUtterances, actualUtterances, options.CompareText);
```

Comment:
>TestCaseSource [](start = 37, length = 14)

should this rather be a utils class instead of calling what seems to be test code from the commande?